kolibri/deployment/default/settings/base.py

```diff
@@ -115,6 +116,9 @@
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': os.path.join(KOLIBRI_HOME, 'db.sqlite3'),
+        'OPTIONS': {
+            'timeout': 100,
```

Comment:
Needed in order to avoid timeouts during massive DB transfers that are done as a single snapshot transaction.